test/e2e/functional_test.go

```diff
@@ -162,6 +162,17 @@ func (s *FunctionalSuite) TestEventOnPVCFail() {
 		})
 }
 
+func (s *FunctionalSuite) TestDefaultPDB() {
+	s.Given().Workflow("@functional/default-pdb-support.yaml").When().SubmitWorkflow().Wait(10 * time.Second).
+		Then().ExpectWorkflow(func(t *testing.T, meta *metav1.ObjectMeta, status *wfv1.WorkflowStatus) {
+		pdb, _ := s.KubeClient.PolicyV1beta1().PodDisruptionBudgets("argo").Get(meta.Name+"-pdb", metav1.GetOptions{})
+		assert.NotNil(t, pdb)
+		time.Sleep(40 * time.Second)
```

Comment:
this is a bit too long to sleep in a test - it brings you test immediately to the top of this list for culling when the suites gets too long.

Can you please reduce this time?

Reply:
I think you should use assert.NoError